README.md

```diff
@@ -7,7 +7,7 @@ This fixes that and also makes it work for all Discord versions.
 
 ## Demo
 
-![Demo GIF of the improved installer](./installer/demo.gif)
+[![asciicast](https://asciinema.org/a/Hr6o4z5O2AMEZKnTTPQenpOot.svg)](https://asciinema.org/a/Hr6o4z5O2AMEZKnTTPQenpOot)
```

Comment:
Shouldn't this be `https://asciinema.org/a/594421?autoplay=1` to enable autoplay on open?

Reply:
sure

Reply:
I think I'd rather use [this](https://alukach.com/posts/animated-terminal-output/) so an animation plays for viewers, this could just be wrapped in a `<a>`/markdown link to the asciicast.

Reply:
Wow, that's really nice! On it right now